bevy_kayak_ui/src/render/unified/shader.wgsl

```diff
@@ -51,39 +51,26 @@ var image_sampler: sampler;
 
 let RADIUS: f32 = 0.1;
 
-fn sd_box_rounded(
-    frag_coord: vec2<f32>,
-    position: vec2<f32>,
-    size: vec2<f32>,
-    radius: f32,
-) -> f32 {
-    var inner_size: vec2<f32> = size - radius * 2.0;
-    var top_left: vec2<f32> = position + radius;
-    var bottom_right: vec2<f32> = top_left + inner_size;
-
-    var top_left_distance: vec2<f32> = top_left - frag_coord;
-    var bottom_right_distance: vec2<f32> = frag_coord - bottom_right;
-
-    var dist = max(max(top_left_distance, bottom_right_distance), vec2<f32>(0.0));
-
-    return length(dist);
+fn sdRoundBox(p: vec2<f32>, b: vec2<f32>, r: f32) -> f32 
```

Comment:
Maybe add a comment clarifying what `p`, `b`, and `r` are? I'm not super familiar with shaders so something like that would be helpful if I (or anyone else) need to use this function.

Reply:
P is the position for the given fragment. I use local position since that seems to be the easiest to calculate. B is the size of the rounded box and R is the radius of the circle at the corner.

Reply:
I meant in the code 😅

Reply:
Done!